plugins/serializers/influx/README.md

```diff
@@ -30,6 +30,15 @@ for interoperability.
   ## integer values.  Enabling this option will result in field type errors if
   ## existing data has been written.
   influx_uint_support = false
+
+  ## By default, the line format timestamp is at nanosecond precision. The
+  ## precision can be adjusted here. This parameter can be used to set the
+  ## timestamp units to nanoseconds (`ns`), microseconds (`us` or `µs`),
+  ## milliseconds (`ms`), or seconds (`s`). Note that this parameter will be
+  ## truncated to the nearest power of 10, so if the `influx_timestamp_units`
+  ## are set to `15ms` the timestamps for the serialized line will be output in
+  ## hundredths of a second (`10ms`).
+  influx_timestamp_units = "1ns"
```

Comment:
In a perfect world, I'd like to remove the `1` in `1ns`, but I understand that everything operates on `Duration`s which needs a magnitude. I think my instinct would be to not put a magnitude. It would just fail to deserialize the config, right?

Reply:
Oh yea, I'd say follow the convention if one exists. Didn't realize that this was established

Reply:
It does seem a bit counter intuitive to have a number here. Maybe when we implement the corresponding change on the monitoring-agent side, we could just set the units as `ns`, `s`, `ms` etc and always prepend `1` when writing out the telegraf config.

Reply:
To follow up on this, the number is required. It's unfortunate, but I don't think it should change as there's already precedence from the JSON serializer.